### PR TITLE
Updated 

### PR DESCRIPTION
**Pull request by Basarqari**

### CODE REVIEW

**Comment by dkayiwa** [comment]
Can you squash these commits into one?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,26 @@
...
+
+# Sending a Message
+
+To send an SMS message, use the following code:
+
+This sends the message "Hello, world!" to the number 18007654321 via SMS. If the supplied phone number or message are badly formatted, an exception will be thrown.
+
+Sending a Message to Multiple Recipients
+
+ To send a message to multiple recipients, you must create a Message object and add messaging addresses as recipients. You should retrieve
```
Comment (harsha89) [inline README.md]: YOu should retrieve? Sentence need to be formatted and corrected

File: README.md [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+
+Sending a Message to Multiple Recipients
+
+ To send a message to multiple recipients, you must create a Message object and add messaging addresses as recipients. You should retrieve 
+
+# Listening for Received Messages
+
+ To register a listener that will receive alerts when a message comes in, use this line of code:
+
+The listener must implement the IncomingMessageListener interface.
```
Comment (harsha89) [inline README.md:22]: Please arrange in a proper way

**Comment by Basarqari** [comment]
Of course

**Comment by Basarqari** [comment]
Ignore First four Files, last one is the considerable 
https://github.com/Basarqari/openmrs-module-messaging/commit/58f0a71c7b85d94e35de9ab29ca3d04f2c279b1a

File: README.md [hunk 3]
```diff
@@ -1,6 +1,25 @@
-openmrs-module-messaging
+OpenMRS-Module-Messaging
```
Comment (robbyoconnor) [inline README.md:1]: no. revert this.

File: README.md [hunk 4]
```diff
@@ -1,6 +1,25 @@
...
+
+Sending a Message to Multiple Recipients
+To send a message to multiple recipients, you must create a Message object and add messaging addresses as recipients. You should retrieve 
+
+
+# Listening for Received Messages
+
+To register a listener that will receive alerts when a message comes in, use this line of code:
+The listener must implement the IncomingMessageListener interface.
+
```
Comment (robbyoconnor) [inline README.md:23]: Broken english

**Comment by robbyoconnor** [comment]
This doesn't add much of value

**Pull request closed without merging**